src/app/app.module.ts

```diff
@@ -28,7 +28,7 @@ import { UpdateService } from './core/api/update.service';
 import { ImportedDataService } from './core/utils/imported-data.service';
 import { RequestDisplayComponent } from './components/headers/request-display/request-display.component';
 import { ModalRequestsComponent } from './components/modals/modal-requests/modal-requests.component';
-
+import { TitleCasePipe } from '@angular/common'; 
```

Comment:
A quoi te sert l'import ici ?

Reply:
j'tuilise titlecasepipe our transformer month - per year machin en title case parce que sont définis au sein de ts ( avant y  to Lowercase - toUppercase et du coup y a pas comment faire pour avoir title case sauf utiliser la pipeline c'est tout )

Reply:
Oui mais là il est pas utilisé hors cet import, est ce qu'il est pas supprimable ici ?

Reply:
ah c'est dans update service attends j'ai pas vu XD

Reply:
ah je l'ai mis dans les providers